app.py

```diff
@@ -1,55 +1,72 @@
-from flask import Flask, request
-from flask_restful import reqparse, abort, Api, Resource
-from flask_pymongo import PyMongo
+from flask import Flask, jsonify
+from flask_restful import Resource, Api, reqparse, request, abort
+from model import Model
 
 app = Flask(__name__)
-mongo = PyMongo(app)
 api = Api(app)
 
-
-USERS = {
-    'user1': {'name': 'elena'},
-    'user2': {'name': 'bob'},
-}
-
-def abort_if_user_doesnt_exist(user_id):
-    if user_id not in USERS:
-        abort(404, message="User {} doesn't exist".format(user_id))
-
 parser = reqparse.RequestParser()
 parser.add_argument('name')
 
-class Users(Resource):
-    def get(self, user_id):
-        abort_if_user_doesnt_exist(user_id)
-        return USERS[user_id]
+class Student(Resource):
+    def get(self, student_id):
+        try:
+            student=Model.get_by_id(student_Id)
+        except:
```

Comment:
There are multiple exceptions to be caught here, however 404 would not be a return from this exception. If `None` was returned from the `get_by_id` it would than mean the Application would return a 404